IPCTests/SocketTest.cpp

```diff
@@ -6,7 +6,13 @@
 #define AWAIT_MESSAGE_TIMEOUT    2
 #define AWAIT_CONNECTION_TIMEOUT 3
 
-#define AMOUNT_OF_TESTS 10000
+#ifdef WIN32
+#define DOUBLE_INIT_CODE IPCLIB_SERVER_ERROR
+#endif
+#ifdef __linux__
+#define DOUBLE_INIT_CODE IPCLIB_SUCCEED
+#endif
+#define AMOUNT_OF_TESTS 1
```

Comment:
You now have for-loops over 1 value, either remove the for-loops or increase the amount of tests.

Reply:
correct, this was a test. I will change it back to a bigger number